src/resources/postcss/components/_events-bar.pcss

```diff
@@ -159,6 +159,40 @@
 		width: 21px;
 	}
 
+	.tribe-events-c-events-bar__tabs {
+		display: flex;
+	}
+
+	.tribe-events-c-events-bar__tab {
+		border-radius: var(--border-radius-default);
+		display: flex;
+		flex: 1 1 50%;
+		flex-direction: column;
+		align-items: center;
+		padding: var(--spacer-3) 0;
+
+		&:hover {
+			background-color: var(--color-background-secondary);
+			.tribe-events-c-events-bar__tab-text {
+				font-weight: var(--font-weight-bold);
+			}
+
+		}
+	}
+
+	.tribe-events-c-events-bar__tab-icon {
+		width: var(--spacer-3);
```

Comment:
i don't think we should use spacers here for width and height. they match the px size, but that's not what we're using spacers for. if we decide to double all the spacer sizes but keep the font size, icon size, etc. the same, this would be unintentionally changed. extremely unlikely, but thinking in terms of semantics and maintenance.